### PR TITLE
chore: Update todo.md and small fixes

### PR DESCRIPTION
**Pull request by znikola**
This PR cleans the todo.md file, and applies some small fixes.

### CODE REVIEW

File: feature-libs/checkout/base/core/facade/checkout-delivery-modes.service.spec.ts [hunk 1]
```diff
@@ -212,7 +212,7 @@ describe(`CheckoutDeliveryModesService`, () => {
       );
     });
 
-    // TODO: Replace with event testing once we remove ngrx store.
+    // TODO:#checkout Replace with event testing once we remove ngrx store.
```
Comment (bgambocjaviniar) [inline feature-libs/checkout/base/core/facade/checkout-delivery-modes.service.spec.ts:215]: Shouldn't it be `TODO:#deprecation-checkout` as we're going to remove this after removing ngrx store. Moreover, we're not using #checkout, but instead using #deprecation-checkout

Example: https://github.com/SAP/spartacus/pull/14479/files#diff-be5bc15655bc76f36d02273572519649ee7cafad56caaac87cc7c644010e5185R143

Comment (bgambocjaviniar) [reply feature-libs/checkout/base/core/facade/checkout-delivery-modes.service.spec.ts:215]: this entire file

File: feature-libs/checkout/base/core/facade/checkout-delivery-modes.service.ts [hunk 2]
```diff
@@ -102,7 +102,7 @@ export class CheckoutDeliveryModesService
                     DeliveryModeSetEvent
                   );
                   /**
-                   * TODO: We have to keep this here, since the cart feature is still ngrx-based.
+                   * TODO:#checkout We have to keep this here, since the cart feature is still ngrx-based.
```
Comment (bgambocjaviniar) [inline feature-libs/checkout/base/core/facade/checkout-delivery-modes.service.ts:105]: same case where I mentioned #deprecation-checkout and file being similar to

https://github.com/SAP/spartacus/pull/14479/files#diff-a80b49078c22389d372ba910f61279fd49ae5a3b53651c26807b4ab900e4fd5eR41

**Review by bgambocjaviniar — CHANGES_REQUESTED** [review]

File: feature-libs/checkout/scheduled-replenishment/core/facade/checkout-scheduled-replenishment.service.spec.ts [hunk 3]
```diff
@@ -122,7 +122,7 @@ describe(`CheckoutScheduledReplenishmentService`, () => {
       );
     });
 
-    // TODO: Replace with event testing once we remove ngrx store.
+    // TODO:#checkout Replace with event testing once we remove ngrx store.
```
Comment (bgambocjaviniar) [inline feature-libs/checkout/scheduled-replenishment/core/facade/checkout-scheduled-replenishment.service.spec.ts:125]: same case https://github.com/SAP/spartacus/pull/14479/files#r759590100

File: feature-libs/checkout/scheduled-replenishment/core/facade/checkout-scheduled-replenishment.service.ts [hunk 4]
```diff
@@ -73,7 +73,7 @@ export class CheckoutScheduledReplenishmentService
               tap((replenishmentOrder) => {
                 this.checkoutFacade.setOrder(replenishmentOrder);
                 /**
-                 * TODO: We have to keep this here, since the cart feature is still ngrx-based.
+                 * TODO:#checkout We have to keep this here, since the cart feature is still ngrx-based.
```
Comment (bgambocjaviniar) [inline feature-libs/checkout/scheduled-replenishment/core/facade/checkout-scheduled-replenishment.service.ts:76]: same case here https://github.com/SAP/spartacus/pull/14479/files#r759591608

File: projects/core/src/cart/facade/active-cart.service.spec.ts [hunk 5]
```diff
@@ -173,6 +173,49 @@ describe('ActiveCartService', () => {
     });
   });
 
+  describe('takeActiveCartId', () => {
```
Comment (bgambocjaviniar) [inline projects/core/src/cart/facade/active-cart.service.spec.ts:176]: 👍

File: projects/core/src/cart/facade/active-cart.service.ts [hunk 6]
```diff
@@ -176,8 +176,6 @@ export class ActiveCartService implements OnDestroy {
 
   /**
    * Waits for the cart to be stable before returning the active cart's ID.
-   *
-   * @returns an observable with the active cart ID.
```
Comment (bgambocjaviniar) [inline projects/core/src/cart/facade/active-cart.service.ts:180]: guess we removed it since other methods in this service does not do it for JSdocs right?

**Review by bgambocjaviniar — CHANGES_REQUESTED** [review]

File: feature-libs/checkout/base/core/facade/checkout-delivery-modes.service.ts (hunk 2 above)
Comment (znikola) [reply feature-libs/checkout/base/core/facade/checkout-delivery-modes.service.ts:105]: We will replace this with a facade call, after the cart is merged.

File: feature-libs/checkout/base/core/facade/checkout-delivery-modes.service.spec.ts (hunk 1 above)
Comment (znikola) [reply feature-libs/checkout/base/core/facade/checkout-delivery-modes.service.spec.ts:215]: The `deprecation-checkout` is for the future (see todo.md)

File: feature-libs/checkout/scheduled-replenishment/core/facade/checkout-scheduled-replenishment.service.ts (hunk 4 above)
Comment (znikola) [reply feature-libs/checkout/scheduled-replenishment/core/facade/checkout-scheduled-replenishment.service.ts:76]: Yep, same answer ;)

File: feature-libs/checkout/scheduled-replenishment/core/facade/checkout-scheduled-replenishment.service.spec.ts (hunk 3 above)
Comment (znikola) [reply feature-libs/checkout/scheduled-replenishment/core/facade/checkout-scheduled-replenishment.service.spec.ts:125]: Same answer ;)

File: projects/core/src/cart/facade/active-cart.service.ts (hunk 6 above)
Comment (znikola) [reply projects/core/src/cart/facade/active-cart.service.ts:180]: yeah, we agreed on Slack we'll be reducing the noise, and removing the `@` things in jsdocs.

**Review by bgambocjaviniar — APPROVED** [review]

**Pull request merged**